app/services/service_catalog/rbac_access.rb

```diff
@@ -0,0 +1,32 @@
+module ServiceCatalog
+  class RBACAccess
+    attr_reader :acl
+    def initialize(resource, verb)
+      @resource = resource
```

Comment:
`@resource` and `@verb` are not used outside `initialize`. They don't need to be instance variables.

Reply:
@bzwei I need that to do some extra filtering for example a verb of write implies read + write so I would need to filter further based on that.